WindowsDevicePortalWrapper/WindowsDevicePortalWrapper/DefaultDevicePortalConnection.cs

```diff
@@ -151,11 +173,19 @@ public void UpdateConnection(bool requiresHttps)
                     // We take the first, non-169.x.x.x address we find that is not 0.0.0.0.
                     if ((addressInfo.Address != "0.0.0.0") && !addressInfo.Address.StartsWith("169."))
                     {
+                        string port = "";
+                        string[] addressParts = this.Connection.Authority.Split(':');
```

Comment:
I haven't been able to test it yet because I can't seem to get connected over USB. This code does work for Xbox One and other devices when not connecting over USB.